python/ray/worker.py

```diff
@@ -367,17 +367,17 @@ def put_object(self, object_id, value):
             # should return an error code to the caller instead of printing a
             # message.
             logger.info(
-                "The object with ID {} already exists in the object store."
-                .format(object_id))
+                "The object with ID {} already exists in the object store.".
+                format(object_id))
```

Comment:
Don't use `format` in logging, use `logger.info("... {} ...", arg)` for lazy formatting.